GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -660,11 +660,11 @@ private void RefreshSelection()
             FillFileTree(selectedRevision);
             FillDiff(selectedRevisions);
 
-            string oldBody = selectedRevision?.Body;
+            string oldBody = selectedRevision?.GetBodyAndNotes();
```

Comment:
❗ Previously this call was cheap and idempotent - i.e., accessing the backing field. Now, it's significantly more expensive, which will have a perf effect.

Reply:
This call is in "FormBrowse.RefreshSelection" which is never in any tight, preformance critical loop, as R# suggests. 
Plus this suggestion is in contrafiction to gerhardol`s earlier comments. 
https://github.com/gitextensions/gitextensions/pull/11417#discussion_r1412838976
Memory vs performance, as always. 
Please make up your minds :)

Reply:
It is not like we have discussions at the coffeemachine about this...
I prefer to not allocate here. But RussKie is the boss.

Reply:
Almost all calls (also if Notes are enabled) will return Body without any processing.
So performance impact is minimal.